src/syntax/specification/specification.ts

```diff
@@ -24,11 +24,14 @@ let _elementSpecification: {
  * Registers a syntax element specification from a given specification entry data.
  * @param name name of the syntax element
  * @param specification specification entry data
+ * @returns false if element name already exists, else true
```

Comment:
\`false\` if element name already exists else \`true\`

add the backticks; slight quirk